server/controllers/questionsControllers.js

```diff
@@ -7,25 +7,24 @@ const Question = {
     const data = req.body;
     const theUser = usersModels.signUsers.find(user => user.userId === data.user);
```

Comment:
The variable name theUser doesn't tell me much about what is being stored. You can read this blog post for additional insight. https://medium.com/coding-skills/clean-code-101-meaningful-names-and-functions-bf450456d90c

Reply:
Thank you for the blog post. Would implement the lessons learnt now